### PR TITLE
Improve breadcrumbs on Application (for applicants)

### LINKED ISSUE #5368: Bug - Application Step breadcrumbs are always in English
# Description
Even when viewing page in french, application step breadcrumbs are in english.
Check with @Jerryescandon or @bitliege on what breadcrumbs would be most useful.

### PR DESCRIPTION
**Pull request by tristan-orourke**
Resolves #5368

### CODE REVIEW

**Comment by tristan-orourke** [comment]
This was much easier when I remembered we already have the `custom_breadcrumbs` mechanic 😅

File: app/Http/Controllers/ApplicationTimelineController.php [hunk 1]
```diff
@@ -56,40 +65,41 @@ public function show(JobApplication $jobApplication, string $requestedStep = nul
             ) {
                 return view('applicant/application-timeline-root')
                 ->with([
-                    'title' => $jobApplication->job_poster->title, // TODO: Check with design what the title should be.
+                    'title' => $application->job_poster->title, // TODO: Check with design what the title should be.
```
Comment (mnigh) [inline app/Http/Controllers/ApplicationTimelineController.php]: Can the comment, `TODO: Check with design what the title should be.`,  be removed at this point?

File: app/Http/Controllers/ApplicationTimelineController.php [hunk 2]
```diff
@@ -39,11 +39,20 @@ public function show(JobApplication $jobApplication, string $requestedStep = nul
...
+            $application->job_poster->title => route('jobs.summary', ['jobPoster' => $application->job_poster]),
+            Lang::get('common/breadcrumbs.application_id', ['id' => $application->id]) => '',
+        ];
+
+
         if ($requestedStep === 'welcome') {
             return view('applicant/application-timeline-root')
                 ->with([
-                    'title' => $jobApplication->job_poster->title, // TODO: Check with design what the title should be.
+                    'title' => $application->job_poster->title, // TODO: Check with design what the title should be.
```
Comment (mnigh) [inline app/Http/Controllers/ApplicationTimelineController.php]: Can the comment, `TODO: Check with design what the title should be.`,  be removed at this point?

File: resources/lang/fr/common/breadcrumbs.php [hunk 3]
```diff
@@ -21,6 +21,7 @@
     'job-title-missing' => "'Titre manquant'",
     'review' => 'Revue',
     'builder' => "Constructeur d'affiches",
+    'application_id' => 'Application Id : :id',
```
Comment (mnigh) [inline resources/lang/fr/common/breadcrumbs.php]: Is _Application Id_ the FR translation of the EN _Application Id_? If not, this should be replaced with its FR version.

**Review by mnigh — CHANGES_REQUESTED** [review]
A few very small things to address, but otherwise (in honour of St. Patrick's Day), maith thú!

File: resources/lang/fr/common/breadcrumbs.php (hunk 3 above)
Comment (tristan-orourke) [reply resources/lang/fr/common/breadcrumbs.php]: Its what I got from DeepL translate. I wanted to run it by the team at demo today.

**Review by mnigh — CHANGES_REQUESTED** [review]
Changing the value of the FR string uncovered an issue with [string formatting](https://github.com/GCTC-NTGC/TalentCloud/pull/5446/files#diff-763bb5a579199054b61beac459daae7c765a07906fcd6da4e4e9dff788973b15L11-L19) where it uses Twig's title filter to titlecase the value, which is not FR-friendly.

Screen Shot 2021-03-17 at 15 09 23

**Review by mnigh — APPROVED** [review]
👍

**Pull request merged**